jobsdb/jobsdb.go

```diff
@@ -989,17 +1117,20 @@ func (jd *HandleT) storeJobsDS(ds dataSetT, copyID bool, retryEach bool, jobList
 	var err error
 
 	//Using transactions for bulk copying
-	txn, err := jd.dbHandle.Begin()
-	jd.assertError(err)
+	isTxnPassed := txn != nil
+	if !isTxnPassed {
+		txn, err = jd.dbHandle.Begin()
+		jd.assertError(err)
+	}
 
 	errorMessagesMap = make(map[uuid.UUID]string)
 
 	if copyID {
-		stmt, err = txn.Prepare(pq.CopyIn(ds.JobTable, "job_id", "uuid", "parameters", "custom_val",
+		stmt, err = txn.Prepare(pq.CopyIn(ds.JobTable, "job_id", "uuid", "user_id", "parameters", "custom_val",
```

Comment:
since this also goes in the open source repo, existing tables will not have user_id column. wouldn't this fail in such cases or are we going to somehow run some kind of db migration and add user_id column to existing tables also